modules/ting_search_carousel/ting_search_carousel.install

```diff
@@ -19,3 +20,37 @@ function ting_search_carousel_uninstall() {
   variable_del('ting_carousel_search_queries_hidden');
   variable_del('ting_carousel_search_queries');
 }
+
+function ting_search_carousel_add_shortcut() {
+  // Add shortcut to carousel.
+  $shortcut_set = shortcut_set_load(SHORTCUT_DEFAULT_SET_NAME);
+  $shortcut_set->links = array(
+    array(
+      'link_path' => 'admin/config/ting/ting_search_carousel',
+      'link_title' => st('Configure carousel'),
+      'weight' => -14,
+    ),
+  );
+
+  shortcut_set_save($shortcut_set);
+}
+
+/**
+ * Remove previously added shortcut and add shortcut to carousel again.
+ */
+function ting_search_carousel_update_7001() {
+  db_delete('menu_links')
+    ->condition('link_path', 'admin/config/ting/ting_search_carousel')
+    ->execute();
```

Comment:
As I see it this will also delete the link from the management menu. Is this intentional?